api/portal-api/portal_api.go

```diff
@@ -129,7 +129,7 @@ func (a *PortalLinkHandler) retrievePortalLink(r *http.Request) (*datastore.Port
 
 	if util.IsStringEmpty(token) {
 		cred, _ := middleware.GetAuthFromRequest(r)
-		token = cred.Token
+		token = cred.Token // TODO(all): this should probably be cred.APIKey, token is used for jwt see  middleware.GetAuthFromRequest(r)
```

Comment:
Hmm, does this mean that we would need to pass the token in a different header?

Reply:
No, we found this fix.